frontend/src/modules/EditZing/Components/StudentCard.tsx

```diff
@@ -79,10 +106,12 @@ const StudentCard = ({
                 disableFocusListener
                 disableTouchListener
                 title={
-                  'Requested study partner ' +
-                  (submissionTime.getMonth() + 1) +
-                  '/' +
-                  submissionTime.getDate()
+                  <Typography sx={{ fontWeight: '500', fontSize: '13px' }}>
+                    Requested:{' '}
+                    <Typography sx={{ fontWeight: '900', fontSize: '18px' }}>
+                      {submissionTime.getMonth() + 1}/{submissionTime.getDate()}
+                    </Typography>
```

Comment:
Not gonna lie, the first time I saw this I was confused because I thought it said 7/6 as in seven sixths or like score 7 out of 6 instead of a date.

I think consistency with how Miranda did the tooltips for the group dot timestamps and the designs for the tooltips would be better (current tooltips for groups and students use default MUI styling. Perhaps add a style override in the theme to make all the tooltips look the same?)

Reply:
Yeah I mainly changed it because sometimes the tooltip would get on top of the edit notes icon. But I've changed it back to what it was.